util/workers.py

```diff
@@ -28,6 +28,10 @@ def get_worker_count(worker_kind_name, multiplier, minimum=None, maximum=None):
 
     # Check for an override via an environment variable.
     override_value = os.environ.get("WORKER_COUNT_" + worker_kind_name.upper())
+
+    if override_value:
+        override_value = int(override_value)
```

Comment:
nit: You're already checking override_value below. Also check for a `ValueError` when doing the `int` conversion.

Reply:
I could handle the potential `ValueError` but want it to fail-fast if Quay is deployed with a non-integer.

I'm not opposed to giving a better error message though. Thoughts @kleesc ?